app/views/layouts/_footer.html.erb

```diff
@@ -7,16 +7,11 @@
         <% unless Skin.in_chooser.empty? %>
           <li class="module group">
             <h4 class="heading"><%= ts('Customize', key: 'footer') %></h4>
-            <ul class="menu">
-              <li aria-haspopup="true">
-                <a href="#"><%= ts('Choose Skin', key: 'footer') %></a>
-                <ul id="skin_chooser" class="expandable secondary">
-                  <li><%= link_to ts('Default', key: 'footer'), unset_skins_path %></li>
-                  <% Skin.in_chooser.order(:title).each do |s| %>
-                    <li><%= link_to s.title, set_skin_path(s) %></li>
-                  <% end %>
-                </ul> 
-              </li>
+            <ul id="skin_chooser" class="menu">
```

Comment:
Can we get rid of the `id` here? Removing it may require updating some tests, but we try not to use IDs unless they're necessary for things other than tests/styling (e.g., JavaScript).